### ISSUE TITLE
Unable to get numbers chars

### ISSUE THREAD

**Issue by joaovictorrx** [smalot/pdfparser#523]
Could not get numbers content of this file
[boleto-caixa.pdf](https://drive.google.com/file/d/1yDEWmaFCOmzTQYAhSHCX9FrRb3eJtYDs/view?usp=sharing)
Any help?

**Comment by joaovictorrx** [comment]
If help in something, using getDetails() in this files we get:

[Creator] => Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.82 Safari/537.36

[Producer] => Skia/PDF m99

**Comment by k00ni** [comment]
> Any clue for this problem?

No. I don't have time to help you here sorry. You will have to wait until someone else work on this.
